src/Powercord/plugins/pc-titleBarGames/index.js

```diff
@@ -45,11 +45,12 @@ module.exports = class TitleBarGames extends Plugin {
         })
       );
 
-    applications.push(React.createElement(this.icon, {
-      name: 'Library',
-      className: `pc-game-img ${this.iconClass.linkButtonIcon}`,
-      onClick: () => this.navigator.transitionTo(Routes.APPLICATION_LIBRARY)
-    }));
+    applications.push(
+      React.createElement('div', {
+        className: `pc-game-img ${this.iconClass.linkButtonIcon}`,
```

Comment:
As I said on a previous PR, `.pc-` is usually reserved to normalized classes and for custom ones plugins usually uses `.powercord-`. If you can update it in this PR that'd be awesome :)

Reply:
this is from your plugin but i can update if you wish

Reply:
![200iq](https://user-images.githubusercontent.com/14052956/55832773-321de300-5b0e-11e9-842c-4a68202c6ec1.gif)

Reply:
done, can't test as it titleBarGames uh .. crashes my discord. plz fix